### ISSUE TITLE
Ourgroceries lovelace card malfunctioning on latest HA (2022.4.0)

### ISSUE THREAD

**Issue by lxqikrana** [ljmerza/ha-our-groceries#27]
## Version of the custom_component
Our Groceries: 1.3.12
Card: 1.6.1

## Configuration

```
ourgroceries:
  username: !secret our_groceries_username
  password: !secret our_groceries_password

Card config:
type: custom:our-groceries-card
title: Shopping List
entity: sensor.our_groceries
show_crossed_off: true
sort_by_name: false
hide_empty_lists: false
```

## Describe the bug
The ourgroceries card just shows an empty card without any of my lists. It was working on HA version 2022.3.8, stopped working on 2022.4.0
![image](https://user-images.githubusercontent.com/12616210/162210488-005d76a6-f36c-4417-9b73-aecace5821a4.png)

## Debug log

The only thing related to ourgroceries in the log is this one line:

```text

2022-04-07 08:21:13 WARNING (MainThread) [homeassistant.setup] Setup of ourgroceries is taking over 10 seconds.

```

**Comment by lxqikrana** [comment]
Switched back to 2022.3.8 and the card works fine:
![image](https://user-images.githubusercontent.com/12616210/162218027-66a32aac-63cb-4d30-8b9a-055734465647.png)

**Comment by ljmerza** [comment]
fixed in v1.3.13
